data_collection/gazette/spiders/pa_cameta.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+from urllib.parse import urlencode
+
+import re
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+RE_EDITION_NUMBER = re.compile(r"Edi..o\s*(\d+)")
+RE_EXT_DATE = re.compile(r"\s+(\d+)\sde\s(\w+)\sde\s(\d+)")
+
+
+class PaCameta(BaseGazetteSpider):
+    TERRITORY_ID = "1502103"
+    name = "pa_cameta"
+    allowed_domains = ["prefeituradecameta.pa.gov.br"]
+    start_date = datetime.date(2023, 1, 12)
+    download_file_headers = {"Accept": "application/octet-stream"}
+
+    BASE_URL = "https://prefeituradecameta.pa.gov.br/diario-oficial-do-municipio/"
+
+    def start_requests(self):
+        url = f"{self.BASE_URL}"
+        yield scrapy.Request(url, callback=self.parse)
+
+    def parse(self, response):
+        a_elements = response.css("#post-20884 > div > ul > li > a")
+        for a in a_elements:
+            text = a.css("a::text").get()
+            href = a.css("a::attr(href)").get()
+            # This link seems to be a test. will be ignored
+            if "12 de abril de 2022" in text:
+                continue
+            gazette_edition_number = self.get_edition_number(text)
+            gazette_date = self.get_date(text)
+            gazette_url = href
+            if gazette_date >= self.start_date and gazette_date <= self.end_date:
```

Comment:
Essa condicional pode ser simplificada.
```suggestion
            if self.start_date <= gazette_date <= self.end_date:
```